tools/zbkey

```diff
@@ -173,24 +183,20 @@ print "Sending data request packet..."
 #print kbutils.hexdump(data_assoc_packet)
 kb.inject(data_assoc_packet)
 
-# receiving the association response and setting the device_short_address
-value = None
-start = time.time()
-while (start+arg_delay > time.time()):		
-    # Does not block
+
+while True:
```

Comment:
This seems to remove the arg_delay command line parameter from being used. I suggest we keep that in there for users who expect it unless you see a reason to remove it?